Adjust telepathy-glib.pc to our version of pkg-config
diff --git a/telepathy-glib/telepathy-glib.pc.in b/telepathy-glib/telepathy-glib.pc.in
index e2ddf03..6038011 100644
--- a/telepathy-glib/telepathy-glib.pc.in
+++ b/telepathy-glib/telepathy-glib.pc.in
@@ -6,7 +6,6 @@ includedir=@includedir@
 Name: Telepathy-GLib
 Description: GLib utility library for the Telepathy framework
 Version: @VERSION@
-Requires: pkg-config >= 0.21
-Requires.private: dbus-glib-1 >= 0.82, glib-2.0 >= 2.24, gobject-2.0 >= 2.24, gio-2.0 >= 2.24
+Requires: pkg-config >= 0.21 dbus-glib-1 >= 0.82, glib-2.0 >= 2.24, gobject-2.0 >= 2.24, gio-2.0 >= 2.24
 Libs: -L${libdir} -ltelepathy-glib
 Cflags: -I${includedir}/telepathy-1.0
